catboost/python-package/catboost/helpers.h

```diff
@@ -16,11 +16,16 @@
 #include <catboost/libs/train_lib/options_helper.h>
 
 #include <library/cpp/json/json_value.h>
+#include <library/cpp/threading/local_executor/tbb_local_executor.h>
 
 #include <util/generic/array_ref.h>
 #include <util/generic/fwd.h>
 #include <util/generic/noncopyable.h>
+#include <util/generic/ptr.h>
 #include <util/generic/xrange.h>
+#include <util/system/guard.h>
+#include <util/system/info.h>
+#include <util/system/mutex.h>
```

Comment:
looks like this staff needs to go to helpers.cpp, since it is not used here

how did you benchmark this improvement?

please change the name of this pr to `Cache TBB local executors` -- it is more clear

Reply:
> looks like this staff needs to go to helpers.cpp, since it is not used here

done

> how did you benchmark this improvement?

there is another pr (#2185) with a benchmark

> please change the name of this pr to Cache TBB local executors -- it is more clear

done

Reply:
Done